src/pokemon_jump_4.c

```diff
@@ -0,0 +1,922 @@
+    window.height = 4;
+    window.paletteNum = 2;
+    window.baseBlock = 0x2B;
+
+    CreateYesNoMenu(&window, 2, 0, 2, 0x00a, 0xD, a);
+}
+
+static void sub_814A84C(void)
+{
+    u8 color[] = {TEXT_COLOR_TRANSPARENT, TEXT_COLOR_DARK_GREY, TEXT_COLOR_LIGHT_GREY};
+
+    PutWindowTilemap(0);
+    PutWindowTilemap(1);
+    FillWindowPixelBuffer(0, 0);
+    FillWindowPixelBuffer(1, 0);
+    AddTextPrinterParameterized3(0, 0, 0, 2, color, 0, gText_SpacePoints2);
+    AddTextPrinterParameterized3(1, 0, 0, 2, color, 0, gText_SpaceTimes3);
+}
+
+static const u8 gUnknown_846D953[] = {2, 2, 0, 0, 1, 1, 1, 0, 0, 2, 0, 0, 0};
+
+static const struct CompressedSpriteSheet gUnknown_846D960 = {gUnknown_8479688, 0, 0x320};
+static const struct SpritePalette gUnknown_846D968 = {gUnknown_8479668, 0x320};
+
+static const u16 gUnknown_846D970[] = {0x06, 0x08, 0x10, 0x08};
+static const u16 gUnknown_846D978[] = {0x06, 0x08, 0x0b, 0x06, 0x10, 0x08};
+static const u16 gUnknown_846D984[] = {0x02, 0x06, 0x06, 0x08, 0x10, 0x08, 0x14, 0x06};
+static const u16 gUnknown_846D994[] = {0x02, 0x06, 0x06, 0x08, 0x0b, 0x06, 0x10, 0x08, 0x14, 0x06};
+
+static const u16 *const gUnknown_082FE234[] =
+{
+    gUnknown_846D970,
+    gUnknown_846D978,
+    gUnknown_846D984,
+    gUnknown_846D994,
+};
+
+static const s16 gUnknown_846D9B8[] = {0x0058, 0x0098};
+static const s16 gUnknown_846D9BC[] = {0x0058, 0x0078, 0x0098};
+static const s16 gUnknown_846D9C2[] = {0x0038, 0x0058, 0x0098, 0x00b8};
+static const s16 gUnknown_846D9CA[] = {0x0038, 0x0058, 0x0078, 0x0098, 0x00b8};
+
+static const s16 *const gUnknown_846D9D4[] =
+{
+    gUnknown_846D9B8,
+    gUnknown_846D9BC,
+    gUnknown_846D9C2,
+    gUnknown_846D9CA,
+};
+
+static void sub_814A8B8(void)
+{
+    int i, y, playersCount = sub_81499A4();
+    const s16 *xCoords = gUnknown_846D9D4[playersCount - 2];
+
+    for (i = 0; i < playersCount; i++)
+    {
+        struct PokemonJump1_MonInfo *info = sub_81499C0(i);
+
+        y = gMonFrontPicCoords[info->species].y_offset;
+        sub_814ADCC(gUnknown_203F3D8, info, *xCoords, y + 112, i);
+        sub_814B240(gUnknown_203F3D8, *xCoords, 112, i);
+        xCoords++;
+    }
+}
+
+void sub_814A940(u32 id, s16 y)
+{
+    gUnknown_203F3D8->unk81A8[id]->pos2.y = y;
+}
+
+void sub_814A95C(int id)
+{
+    sub_814B348(gUnknown_203F3D8, id);
+    ChangeBgY(2, (gUnknown_846D953[id] * 5) << 0xD, 0);
+}
+
+int sub_814A98C(u8 flags)
+{
+    int i, count;
+
+    for (i = 0, count = 0; i < 5; i++)
+    {
+        if (flags & 1)
+        {
+            sub_814AF0C(gUnknown_203F3D8, i);
+            count++;
+        }
+        flags >>= 1;
+    }
+
+    sub_814ACCC(count - 2);
+    return count;
+}
+
+static void sub_814A9C8(void)
+{
+    struct DigitObjUtilTemplate unkStruct;
+    struct DigitObjUtilTemplate *ptr = &unkStruct; // This temp variable is needed to match, don't ask me why.
+
+    ptr->shape = SPRITE_SHAPE(8x8);
+    ptr->size = SPRITE_SIZE(8x8);
+    ptr->strConvMode = 0;
```

Comment:
Does `STR_CONV_MODE_LEFT_ALIGN` apply here?

Reply:
It doesn't look to be the same as with ConvertIntToDecimalStringN